### ISSUE TITLE
[Linux-tkg patch request]  Josh Ashton Kernel Patches for HDR gaming

### ISSUE THREAD

**Issue by 123-123-nil** [Frogging-Family/community-patches#143]
According to this Arch Wiki article (https://wiki.archlinux.org/title/HDR_monitor_support#HDR_gaming_support), it appears that certain patches are necessary to enable HDR support in Gamescope. These patches are available in the patched kernel source, which can be found here: https://gitlab.freedesktop.org/JoshuaAshton/linux-hdr/-/tree/josh-hdr-colorimetry/.

It would be great if these patches could be included in this repository, as it would make it easier for users to experiment with HDR on Linux.

**Comment by ThisNekoGuy** [comment]
I also agree with this, it would be great given that there is currently know other way to use HDR besides using specific non-tkg kernels.

**Comment by ptr1337** [comment]
https://github.com/CachyOS/kernel-patches/blob/master/6.5/0001-amd-hdr.patch

Tested against 6.5
